src/main/java/teammates/client/remoteapi/RemoteApiClient.java

```diff
@@ -8,21 +8,31 @@
 import teammates.test.driver.TestProperties;
 
 public abstract class RemoteApiClient {
-
+    private final String LOCALHOST = "localhost";
+    
     protected void doOperationRemotely() throws IOException {
         TestProperties testProperties = TestProperties.inst();
 
+        String appDomain = testProperties.TEAMMATES_REMOTEAPI_APP_DOMAIN;
+        int appPort = testProperties.TEAMMATES_REMOTEAPI_APP_PORT;
+        
         System.out.println("--- Starting remote operation ---");
-        System.out.println("Going to connect to:"
-                + testProperties.TEAMMATES_REMOTEAPI_APP_DOMAIN + ":"
-                + testProperties.TEAMMATES_REMOTEAPI_APP_PORT);
-
-        RemoteApiOptions options = new RemoteApiOptions().server(
-                testProperties.TEAMMATES_REMOTEAPI_APP_DOMAIN,
-                testProperties.TEAMMATES_REMOTEAPI_APP_PORT).credentials(
-                testProperties.TEST_ADMIN_ACCOUNT,
-                testProperties.TEST_ADMIN_PASSWORD);
-
+        System.out.println("Going to connect to:" + appDomain + ":" + appPort);
+
+        RemoteApiOptions options = new RemoteApiOptions().server(appDomain, appPort);
+
+        boolean isDevServer = appDomain.equals(LOCALHOST);
+        if (isDevServer) {
+            // Dev Server doesn't require credential.
+            options.useDevelopmentServerCredential();
+        } else {
+            // If you are trying to run script on Staging Server:
```

Comment:
I think we will have to move these instructions to somewhere more formal now then, possibly the setting up guide.